WebDriverAgentLib/Categories/XCElementSnapshot+FBHelpers.m

```diff
@@ -97,11 +99,14 @@ - (XCElementSnapshot *)fb_parentMatchingType:(XCUIElementType)type
   return snapshot;
 }
 
-- (XCElementSnapshot *)fb_parentMatchingOneOfTypes:(NSArray<NSNumber *> *)types
+- (XCElementSnapshot *)fb_findVisibleParentMatchingOneOfTypesAndHasMoreThanOneChild:(NSArray<NSNumber *> *)types
```

Comment:
is it fine if you take this in for now? Anyways you are going to refactor entire scrolling logic.